beacon-chain/rpc/eth/v1/validator/validator_test.go

```diff
@@ -305,6 +314,97 @@ func TestGetProposerDuties_SyncNotReady(t *testing.T) {
 	assert.ErrorContains(t, "Syncing to latest head, not ready to respond", err)
 }
 
+func TestGetBlock(t *testing.T) {
```

Comment:
This test is a slightly modified copy of https://github.com/prysmaticlabs/prysm/blob/526596a679c60ead7fe1fdbc4f136668ecd2d1d2/beacon-chain/rpc/prysm/v1alpha1/validator/proposer_test.go#L39